### PR TITLE
Feat(CI): tag team on failed workflow

### PR DESCRIPTION
**Pull request by saber-solooki**
https://adjustcom.atlassian.net/browse/AI-279

We want to remove `@here` in our general channels. From now on, every services needs to update their `CI` and specify their team id by using `team_id` field. So in case of having failed workflow, just related team will be notified. If this field is not provided, all channels member will be notified by `@here`.

Example:
```
check:
    uses: adjust/githubWorkflows/.github/workflows/automateChecks.yml@ci-improvement
    needs: [get-python-version]
    strategy:
      matrix:
        check: ['make flake8', 'make mypy', 'make bandit', 'make check-licenses', 'make test']
    with:
      check: ${{ matrix.check }}
      custom_db_connect_args: '{"server_settings": {"jit": "off"}}'
      python_version: ${{ needs.get-python-version.outputs.python_version }}
      team_id: S04Q40HRCHY
    secrets:
      PIP_SSH_KEY: ${{ secrets.AUTOMATE_PIP_SSH_KEY }}
      SLACK_WEBHOOK_URL: ${{ secrets.AUTOMATE_SLACK_WEBHOOK }}
      GITHUB_ORGANIZATION_ACCESS_TOKEN: ${{ secrets.AUTOMATE_GITHUB_ACCESS_TOKEN }}
```

In our services, we would need `team_id` to be specified in `automateChecks` and `automateDeploy`.
In order to find your team_id, just open slack in your browser and your team_id would be at the end of the link
Untitled 2
or you can follow this [link](https://api.slack.com/reference/surfaces/formatting#mentioning-groups)

Example PR for a service:
https://github.com/adjust/configs-service/pull/938

If one team wants to send the notification to another channel, they can update their own workflow by using the same library like this:
```
- name: Send Another Slack notification
  if: always()
  uses: act10ns/slack@v2
  env:
    ANOTHER_SLACK_WEBHOOK_URL: ${{ secrets.ANOTHER_SLACK_WEBHOOK_URL }}
  with:
    status: ${{ job.status }}
```

Besides, we can get rid of our custom fork [here](https://github.com/adjust/github-actions-slack) since the base repo has already supported our custom notification

### CODE REVIEW

**Comment by saber-solooki** [comment]
config file is not taken effect.
I'm not sure if I get the answer correctly: https://github.com/act10ns/slack/issues/257
Do we need to merge it to see the result?

File: .github/workflows/automateChecks.yml [hunk 1]
```diff
@@ -110,10 +114,12 @@ jobs:
...
+#        if: github.event_name == 'push' && failure()
+        if: failure()
+        uses: act10ns/slack@v2
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          TEAM_TAG: ${{ inputs.team_tag }}
         with:
           status: ${{ job.status }}
-          steps: ${{ toJson(steps) }}
+          config: .github/workflows/slack_template.yml
```
Comment (eduardostalinho) [inline .github/workflows/automateChecks.yml:125]:
```suggestion
          config: .github/workflows/templates/slack_template.yml
```

File: .github/workflows/mfeContinuousIntegration.yml [hunk 2]
```diff
@@ -383,15 +388,13 @@ jobs:
 
       - name: Send Slack notification
         if: ${{ failure() && needs.configure_variables.outputs.e2e_slack_enabled == 'true' }}
-        uses: adjust/github-actions-slack@v1.2.5
+        uses: act10ns/slack@v2
```
Comment (eduardostalinho) [inline .github/workflows/mfeContinuousIntegration.yml:391]: 🙏

**Review by eduardostalinho — APPROVED** [review]
Do we have a test run that uses this branch?

File: .github/workflows/automateChecks.yml [hunk 3]
```diff
@@ -110,10 +114,12 @@ jobs:
         if: inputs.check == 'make check-licenses'
 
       - name: Send Slack notification
-        if: github.event_name == 'push' && failure()
-        uses: adjust/github-actions-slack@v1.2.5
+#        if: github.event_name == 'push' && failure()
```
Comment (eduardostalinho) [inline .github/workflows/automateChecks.yml:117]: Don't forget to uncomment this line before the merge.

**Comment by saber-solooki** [comment]
> Do we have a test run that uses this branch?

Yes. [CS](https://github.com/adjust/configs-service/pull/934)

**Pull request merged**